docs/adrs/00005-license-export-analytics.md

```diff
@@ -0,0 +1,126 @@
+```
+If I understand correctly, CycloneDX's SBOM-level License refers to the license of the package corresponding to the SBOM, while SPDX's SBOM-level License refers to the license of the SBOM document itself.
+#### 2 Package-level licenses
+##### 2.1 CycloneDX
+Contains three types: Id, name, and Expression
+The Expression follows the SPDX expression standard.
+```json
+"licenses" : [  {    "license" : {      "id" : "EPL-1.0"    }  },  {    "license" : {      "name" : "GNU Lesser General Public License",      "url" : "http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html"    }  }],`
+
+```
+##### 2.2 SPDX
+It's Expression and follows the SPDX expression standard.
+```json
+"licenseDeclared": "Apache-2.0 OR LicenseRef-GPL-2.0-with-classpath-exception",
+```
+##### 2.3 Special Types in SPDX: ExtractedLicensingInfos
+```json
+"hasExtractedLicensingInfos": [
+{
+"comment": "External License Info is obtained from a build system which predates the SPDX specification and is not strict in accepting valid SPDX licenses.",
+"extractedText": "The license info found in the package meta data is: GPL-2.0-with-classpath-exception. See the specific package info in this SPDX document or the package itself for more details.",
+"licenseId": "LicenseRef-GPL-2.0-with-classpath-exception",
+"name": "GPL-2.0-with-classpath-exception"
+},
+```
+The licenseId can be associated with the license in the package.
+### 2 Current license export requirements
+https://docs.google.com/spreadsheets/d/1Ki9hkLl94L3G4A4rfQQ4wuEnQFTPkpNflClVjlk2mQA/edit?gid=0#gid=0
+
+Here is a simple summary: This function needs to obtain all the packages from an SBOM, then retrieve the licenses of these packages 
+and list this information in a CSV file. Additionally, for SPDX SBOMs, it should also list the information from the ExtractedLicensingInfos 
+in a separate CSV, because this type of SBOM allows License-Refs that reference information from ExtractedLicensingInfos. An important point is that in SPDX SBOMs, 
+there are two types of licenses: declared license and concluded license. The requirement is to only list the declared licenses.
+### 3 Current license data structure design in trustify
+
+```rust
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: Uuid,
+    pub text: String,
+    pub spdx_licenses: Option&lt;Vec&lt;String&gt;&gt;,
+    pub spdx_license_exceptions: Option&lt;Vec&lt;String&gt;&gt;,
+}
+```
+The text field corresponds to CycloneDX's Id, Name, or complete Expression. When it's an Expression, it will be parsed into a license array and stored in spdx_licenses, while parsing exceptions are stored in spdx_license_exceptions.
+For SPDX, the text field corresponds to the complete Expression. Similarly, the Expression will be parsed into a license array and stored in spdx_licenses, with parsing exceptions stored in spdx_license_exceptions.
+Currently missing SPDX's ExtractedLicensingInfos and SBOM-level licenses.
+
+### 4 License data structure in GUAC
```

Comment:
Why is GUAC license structure provided in this ADR ? Please provide the rationale/context of why you are speaking about GUAC here.